gems/codders-dataset/OSVDB-108582.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Then we should support tracking multiple IDs per report. As in, I'm fine with their being only one "vulnerability", but I want to track the multiple OSVDB IDs somehow at least, as otherwise, it's a pain later when matching up OSVDB's DB to ours.